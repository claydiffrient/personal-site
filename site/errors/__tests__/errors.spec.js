/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import { FourOhFour, FiveHundred } from '../index';

describe('Error Tests', () => {
  it('has a proper 404', () => {
    const component = renderer.create(
      <FourOhFour />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('has a proper 500', () => {
    const component = renderer.create(
      <FiveHundred />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
