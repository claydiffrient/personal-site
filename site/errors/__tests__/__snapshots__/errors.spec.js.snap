exports[`Error Tests has a proper 404 1`] = `
<div
  className="mdl-grid root">
  <div
    className="mdl-card mdl-cell mdl-cell--12-col mdl-shadow--4dp">
    <div
      className="mdl-card__title">
      <h2
        className="mdl-card__title-text">
        Page Not Found
      </h2>
    </div>
    <div
      className="mdl-card__supporting-text">
      <p>
        Looks like you were trying to find a page that doesn\'t exist. :(
      </p>
    </div>
  </div>
</div>
`;

exports[`Error Tests has a proper 500 1`] = `
<div
  className="mdl-grid root">
  <div
    className="mdl-card mdl-cell mdl-cell--12-col mdl-shadow--4dp">
    <div
      className="mdl-card__title">
      <h2
        className="mdl-card__title-text">
        Server Error
      </h2>
    </div>
    <div
      className="mdl-card__supporting-text">
      <p>
        Something went horribly wrong, try again later
      </p>
    </div>
  </div>
</div>
`;
